### PR TITLE
[WIP] Meta 3 lakshamana (frontend)

### PR DESCRIPTION
**Pull request by Lakshamana**

### CODE REVIEW

File: src/css/game.css [hunk 1]
```diff
@@ -29,7 +29,7 @@ body {
   line-height: 80px;
   border: 1px solid black
 }
-.oponent-div {
+#oponent-div {
```
Comment (CaioSR) [inline src/css/game.css]: Corrige pra `opponent-div`

File: src/html/game.html [hunk 2]
```diff
@@ -6,24 +6,14 @@
...
-          <div class="board-slot" id="slot-2"></div>
-          <div class="board-slot" id="slot-3"></div>
-          <div class="board-slot" id="slot-4"></div>
-          <div class="board-slot" id="slot-5"></div>
-          <div class="board-slot" id="slot-6"></div>
-          <div class="board-slot" id="slot-7"></div>
-          <div class="board-slot" id="slot-8"></div>
-          <div class="board-slot" id="slot-9"></div>
+          <div id="game-name-rematch">Égua do Jogo Top</div>
+          <div id="oponent-div"> Oponent: XYZ</div>
```
Comment (CaioSR) [inline src/html/game.html]: Corrige oponent pra opponent

File: src/css/game.css (hunk 1 above)
Comment (Lakshamana) [reply src/css/game.css]: resolvido

File: src/html/game.html (hunk 2 above)
Comment (Lakshamana) [reply src/html/game.html]: resolvido

**Pull request merged**